docs/data_source/catalog/iceberg_catalog.md

```diff
@@ -943,17 +977,11 @@ You can specify the table attributes in the `"key" = "value"` format in `propert
    PARTITION BY (id,dt);
    ```
 
-3. Create a partitioned table named `partition_tbl`. The table consists of four columns, `v1`, `v2`, `k1`, and `k2`, of which `k1` and `k2` are defined as partition columns. At the same time, specify that StarRocks synchronously queries an existing table named `employee` and inserts the query result into the new table `partition_tbl`, as shown below:
+3. Query an existing table named `partition_tbl_1`, and create a partitioned table named `partition_tbl_2` into which the query result on `partition_tbl_1` is inserted. For `partition_tbl_2`, `id` and `dt` are defined as partition columns, as shown below:
```

Comment:
Query an existing table named `partition_tbl_1`, and create a partitioned table `partition_tbl_2` based on the query result of `partition_tbl_1`.